script/preCache.js

```diff
@@ -9,17 +9,21 @@ module.exports = breadboard({
     'package.json': require('../package.json')
   },
   entry: ({
+    '/config/index': getConfig,
     '/lib/bundler/index': bundle,
     '/lib/getComponentInfo': getComponentInfo
   }) => {
+    const config = getConfig();
     const components = getComponentInfo().map(componentInfo => componentInfo.name );
+    const preCacheComponentBundles = config.components && config.components.preCacheBundles ? config.components.preCacheBundles : [];
```

Comment:
This line reads the config for the precache bundles which come from the toga.json from components